### DISCUSSION TITLE
Analysis page

### DISCUSSION THREAD

**Discussion by BioMedGPH** [Samedis-care/samedis-care-community#37] [Ideas]
I would like to see some analysis tools/visual representation of the progress/performance of the inventory and inventory work. Either something that is configurable (like pivot tables in MS Excel), or some predefined metrics. These will help to track progress in the performance/efficiency of the maintenance/repair work.

- Monthly number/rolling average of repair tasks created, graph in time.
- Monthly number/rolling average of repair tasks completed, graph in time.
- Monthly number/rolling average of maintenance tasks completed, graph in time. 
- Monthly (rolling) Ratio of Repair over Maintenance tasks, graph in time. Aiming to minimize as good Maintenance would prevent Failures and thus Repairs.
- Average (rolling) time to complete Repair tasks after creating the task, graph in time. Aiming to minize.
- Average (rolling) Percentage of Maintenance completed on time, graph in time. Aiming to maximimze.
- Maybe there is some more to define/thing of.
- Highlighting some equipment that frequently shows malfunction. Not sure how this is best done, maybe calculate the Rolling average  of failures per inventory item over a couple of months and display in time for each inventory item; showing the top 10 or so, or increasing items.

All this data is already in the database, it is just a matter of calculating and presenting it. But very helpful to show progress.

**Comment by YvesR** [comment]
Thanks for your feedback, and we can let you know that we already work on bi.samedis.care, where we can show some analysis charts and lists.
